core/domain/html_validation_service.py

```diff
@@ -703,3 +703,31 @@ def is_parsable_as_xml(xml_string):
         return True
     except xml.etree.ElementTree.ParseError:
         return False
+
+
+def convert_svg_diagram_tags_to_image_tags(html_string):
+    """Renames all the oppia-noninteractive-svgdiagram on the
```

Comment:
See the docstring code style guide here for aligning multiple lines in the initial summary: https://github.com/oppia/oppia/wiki/Coding-style-guide#python.

Reply:
done.